### ISSUE TITLE
problem with josch_sisyphus

### ISSUE THREAD

**Issue by Zhaocka** [floriankagerer/bed-bpp-env#1]
Hi, thank you very much for your meaningful work. I also run the code from [https://github.com/josch/sisyphus](url) , but I can't find out how to modify their dataset to test the effect of their code on our dataset. What do you do here?

**Comment by floriankagerer** [comment]
Hi Zhaocka,
thank your interest in this repo.

If I understand your question correctly, you have problems converting orders in your dataset to a format that can be used with josch_sisyphus?

I did the following steps:
- Convert an order to the format of sisyphus with https://github.com/floriankagerer/bed-bpp-env/blob/main/code/utils/sisyphus/orderConverter.py
(here is a [josch/sisyphus example pallet](https://github.com/josch/sisyphus/blob/master/examples/icra2011/palDay1R1Order.xml))
- Run sisyphus with appropriate arguments (https://github.com/floriankagerer/bed-bpp-env/blob/main/josch_sisyphus/sisyphus-master/run.sh)
- Convert the packlist that was generated of sisyphus to my format https://github.com/floriankagerer/bed-bpp-env/blob/main/code/utils/sisyphus/packlistConverter.py

In summary, to use these converters for more than a single order, I added the bash script 
[createSolutionsWithSisyphus.sh](https://github.com/floriankagerer/bed-bpp-env/blob/main/josch_sisyphus/createSolutionsWithSisyphus.sh)

Did this answer your question?

Best regards,
Florian

P.S.: I am looking forward to your contribution: https://floriankagerer.github.io/leaderboard/  😀

**Comment by Zhaocka** [comment]
ok thank you I will try this
